docker/web-agent/Dockerfile

```diff
@@ -0,0 +1,37 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Start from Java 8 based on Alpine Linux image (~5Mb)
+FROM openjdk:8-jre-alpine
+
+# Settings
+USER root
+ENV AGENT_HOME /opt/ignite/ignite-web-agent
+WORKDIR /opt/ignite
+
+# Add missing software
+RUN apk --update --no-cache add bash
+
+# Copy main binary archive
+COPY ignite-web-agent* ignite-web-agent
+
+# Update permissions
+RUN chmod -R 777 ${AGENT_HOME}
+
+# Entrypoint
+CMD $AGENT_HOME/ignite-web-agent.sh -d ${DRIVER_FOLDER} -n ${NODE_URI} -s ${SERVER_URI} -t ${TOKENS}
```

Comment:
Is it really necessary to change $ {DRIVER_FOLDER}?

Reply:
As we provide a capability to specify this option in run parameters, I think we need to do the same for environment variables. Maybe can we just add default values of env variables? For example:

DRIVER_FOLDER = /opt/ignite/drivers
NODE_URI = http://localhost:8080
SERVER_URI = http://localhost
TOKENS = NO_TOKENS

It should help to avoid some issues when client forgets to init some variables:
http://apache-ignite-users.70518.x6.nabble.com/Agent-not-able-to-connect-tc21714.html#a21735

Reply:
Agree